molecule/driver/openstackdriver.py

```diff
@@ -35,21 +35,17 @@ def __init__(self, molecule):
         self._provider = self._get_provider()
         self._platform = self._get_platform()
         self._openstack = shade.openstack_cloud()
-        self._keypair_name = self.get_keypair_name()
+        self._keypair_name = None
```

Comment:
this seemed like a safer way to init the variable, rather than generating a name we ultimately would overwrite